lesson2/func1.py

```diff
@@ -0,0 +1,10 @@
+# function that calculates the arithmetic average
+if __name__ == '__main__':
+    q = [56, 3, 90, 573]
+    w = int(q[0])
+    e = int(q[1])
+    r = int(q[2])
+    t = int(q[3])
+    def sum(w, e, r, t):
```

Comment:
Declare function outside of main.
Pass the list inside the function and iterate it with for.